src/components/EditionCard/LicenseLink.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Rights } from "~/src/types/DataModel";
+import Link from "~/src/components/Link/Link";
+
+const LicenseLink: React.FC<{ rights: Rights[] }> = ({ rights }) => {
+  return (
+    <Link to="/copyright">
+      {rights && rights.length > 0
+        ? `License: ${rights[0].rightsStatement}`
+        : "License: Unknown"}
```

Comment:
I'm seeing in the BRD that this should be Copyright but it's not reflected in the ticket. I recommend reaching out to Will to see if we want to include that in this change. I commented on the Jira ticket as well.